rust/blockstore/src/config.rs

```diff
@@ -2,6 +2,6 @@ use serde::Deserialize;
 
 #[derive(Deserialize, Debug, Clone)]
 pub enum BlockfileProviderConfig {
-    Arrow(super::arrow::config::ArrowBlockfileProviderConfig),
+    Arrow(Box<super::arrow::config::ArrowBlockfileProviderConfig>),
```

Comment:
Q - Why this change?

Reply:
It keeps the enum variant small.  Clippy started warning about it.

Reply:
Oh, cool. Thanks for explaining